### ISSUE TITLE
Failed to initialize CoreCLR, HRESULT 0x80004005

### ISSUE THREAD

**Issue by CodeJan** [Azure/iotedge#1452]
After running for some time the edgeHub container crashes and won't restart because of CoreCLR error (HRESULT: 0x80004005). 

## Steps to Reproduce
unkown

### Output of `iotedge check`
Command is not available.

### Device (Host) Operating System
Ubuntu 18.04.2 LTS (VM)

### Architecture
amd64

### Container Operating System
Linux containers

#### iotedged
1.0.6.1

#### Edge Agent
1.0

#### Edge Hub
1.0

#### Docker

```
Docker version 3.0.6, build a63faebc
```

## Logs
<details>

<details>
<summary>edge-hub logs</summary>

```
[2019-07-16 11:32:49 : Starting Edge Hub
Failed to initialize CoreCLR, HRESULT: 0x80004005
```
</details>

**Comment by avranju** [comment]
Looking at the .NET Core issues list on GitHub it looks like this error can occur if the .NET runtime is [unable to write to the the temp folder](https://github.com/dotnet/core-setup/issues/4149). To verify if this is the issue do you think you could try adding an environment variable called `COMPlus_EnableDiagnostics` with the value `0` to your Edge Hub deployment and see if this issue occurs?

**Comment by CodeJan** [comment]
I added the COMPlus_EnableDiagnostics variable, now the following error occurs:

![image](https://user-images.githubusercontent.com/14348770/62371545-6c841380-b535-11e9-96cd-0f9e17b4f38d.png)

**Comment by lt72** [comment]
@CodeJan Hello, we believe this issue was fixed with 1.0.8.x release. Please give t a try and your convenience and feel free to re-open this issue as needed.
